shared-bindings/_bleio/__init__.h

```diff
@@ -38,6 +38,16 @@
 
 extern bleio_adapter_obj_t common_hal_bleio_adapter_obj;
 
+extern const mp_obj_type_t mp_type_BluetoothError;
+extern const mp_obj_type_t mp_type_ConnectionError;
+extern const mp_obj_type_t mp_type_RoleError;
+extern const mp_obj_type_t mp_type_SecurityError;
```

Comment:
I think these should be called `mp_type_bleio_ConnectionError`, etc. because they are in the `_bleio` namespace. Note that in CPython, there is a global `ConnectionError`, for instance, and we don't want to potentially conflict with that.